pyproject.toml

```diff
@@ -34,6 +34,7 @@ dependencies = [
     "param",
     "pytz",
     "rubin-scheduler",
+    "lsst.resources",
```

Comment:
I think you're right that lsst.resources will install from pypi, but I'm guessing that for conda-forge it will have to be lsst-resources (and I'd like to ask for a conda-forge package). 
And also, lsst-resources *does* work for pypi as well, so maybe it'd be good to just simply use "lsst-resources" instead of lsst.resources in one place and lsst-resources in another? 
(and I also agree that for pyproject.toml it makes no difference, given how pip checks .. it's entirely for looking 'the same').